src/peep/staff.c

```diff
@@ -372,6 +372,35 @@ void game_command_fire_staff_member(int *eax, int *ebx, int *ecx, int *edx, int
 	*ebx = 0;
 }
 
+void game_command_move_staff_member(int *eax, int *ebx, int *ecx, int *edx, int *esi, int *edi, int *ebp)
+{
+	if (*ebx & GAME_COMMAND_FLAG_APPLY) {
+		int x = *eax;
+		int y = *ecx;
+		int z = *edi;
+
+		uint16 spriteIndex;
+		rct_peep *peep = NULL;
+		FOR_ALL_STAFF(spriteIndex, peep)
```

Comment:
Damn thats what I get for not reading the PR. I don't like the idea of looping through staff_ids we really need to fix sprite index getting out of sync.